.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,7 +21,7 @@ assignees: ''
 
 **Must gather (please complete the following information):**
  - SDK Version [e.g. 1.2.1]
- - Go Version [e.g. Go 1.16.8]
+ - Go Version [e.g. Go 1.17]
```

Comment:
If you change this file here, it's going to be reverted on the next generated PR. Please do this change in the generator repo.

Reply:
Opened PR: https://github.ibm.com/cloudant/sdks/pull/487

Reply:
Reverted in [8110401](https://github.com/IBM/cloudant-go-sdk/pull/245/commits/8110401e5fa063106266677b7c303ed4998de8c9)